docs/Project.toml

```diff
@@ -15,5 +15,5 @@ Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 
 [compat]
-ACEpotentials = "0.6"
+ACEpotentials = "0.8"
```

Comment:
Should there even be a compact entry? Shouldn't the docs build just dev the "current" ACEpotentials?

Reply:
This is the line I get when building docs via CI: 
```
  [3b96b61c] + ACEpotentials v0.8.0-dev `~/work/ACEpotentials.jl/ACEpotentials.jl`
```
So the compat entry in the docs/Project folder is really not needed (and shouldn't be there)